package.json

```diff
@@ -24,7 +24,7 @@
     "dateformat": "^5.0.3",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
-    "express-handlebars": "^7.0.7",
+    "express-handlebars": "^3.1.0",
```

Comment:
uninstalled current handlebars to a "allegedly" working previous version. still does not render.